test/test-custom.conf

```diff
@@ -0,0 +1,47 @@
+# testsuite system configuration
```

Comment:
This file could be shorter since its purpose is just to test the `custom` bootloader backend. Maybe just have two rootfs slots here.